generators/workspace/templates/package.tmpl.json

```
@@ -34,34 +34,34 @@
   },
   "devDependencies": {
     "clean-webpack-plugin": "~3.0.0",
-    "copy-webpack-plugin": "~6.0.3",
-    "css-loader": "~3.6.0",
+    "copy-webpack-plugin": "~6.3.2",
+    "css-loader": "~5.0.1",
     "css-minimizer-webpack-plugin": "^1.1.5",
-    "expose-loader": "~1.0.0",
-    "file-loader": "~6.0.0",
-    "html-loader": "~1.1.0",
-    "html-webpack-plugin": "~4.3.0",
+    "expose-loader": "~1.0.3",
+    "file-loader": "~6.2.0",
+    "html-loader": "~1.3.2",
+    "html-webpack-plugin": "~4.5.0",
     "identity-obj-proxy": "~3.0.0",
     "ifdef-loader": "~2.1.5",
-    "imports-loader": "~1.1.0",
-    "mini-css-extract-plugin": "~0.9.0",
-    "node-sass": "~4.14.1",
+    "imports-loader": "~1.2.0",
+    "mini-css-extract-plugin": "~1.3.2",
+    "node-sass": "~5.0.0",
     "npm-run-all": "~4.1.5",
-    "npm-watch": "^0.6.0",
-    "raw-loader": "~4.0.1",
+    "npm-watch": "^0.7.0",
+    "raw-loader": "~4.0.2",
     "rimraf": "~3.0.2",
-    "sass-loader": "~9.0.2",
-    "shx": "~0.3.2",
-    "source-map-loader": "~1.0.1",
-    "style-loader": "~1.2.1",
-    "tslib": "~2.0.0",
+    "sass-loader": "~10.1.0",
+    "shx": "~0.3.3",
+    "source-map-loader": "~1.1.3",
+    "style-loader": "~2.0.0",
+    "tslib": "~2.0.3",
     "tslint": "~5.20.1",
     "typedoc": "~0.17.8",
     "typescript": "~3.9.7",
-    "url-loader": "~4.1.0",
-    "webpack": "~4.43.0",
-    "webpack-bundle-analyzer": "~3.8.0",
-    "webpack-cli": "~3.3.12",
-    "webpack-dev-server": "~3.10.3"
+    "url-loader": "~4.1.1",
+    "webpack": "~4.44.2",
+    "webpack-bundle-analyzer": "~3.9.1",
```

```suggestion
    "webpack-bundle-analyzer": "~3.9.0",
```

There is no published version 3.9.1 which results in the following error:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for webpack-bundle-analyzer@~3.9.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'ordino_develop'
npm ERR! notarget 
```

Last version published 3.x version is `3.9.0` (see https://www.npmjs.com/package/webpack-bundle-analyzer).

---

Could we even upgrade to the most current version 4?

---

yes, sorry - that was my fault; 4.0 not really because of the acorn problem we had